src/tools/curve_tool_define.cpp

```diff
@@ -37,3 +37,4 @@ DEFINE_string(snapshotCloneDummyPort, "8081", "dummy port of snapshot clone, "
                                     "if specify several, the order should "
                                     "be the same as snapshot clone addr");
 DEFINE_uint64(chunkSize, 16777216, "chunk size");
+DEFINE_bool(useChunkFilePoolAsWalPool, true, "use chunkfilepool as walpool");
```

Comment:
Do not add flag. Try to query it from chunkserver.

Reply:
> Do not add flag. Try to query it from chunkserver.

done